akka-epl/src/main/scala/com/epl/akka_v1/HttpClient.scala

```diff
@@ -0,0 +1,85 @@
+package com.epl.akka_v1
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.marshalling.Marshal
+import akka.http.scaladsl.model.headers.BasicHttpCredentials
+import akka.http.scaladsl.model.{HttpMethods, HttpRequest, RequestEntity, headers}
+import akka.http.scaladsl.unmarshalling.Unmarshal
+import akka.stream.ActorMaterializer
+import akka.util.ByteString
+import com.epl.akka.DocumentType
+
+import scala.concurrent.Future
+
+object HttpClient {
+
+  implicit val system = ActorSystem()
+  implicit val materializer = ActorMaterializer()
+  implicit val ec = system.dispatcher
+
+  final val config = system.settings.config
+
+  private val authorization = headers.Authorization(BasicHttpCredentials(config.getString("cloudant.username"), config.getString("cloudant.password")))
+
+
+  /**
+    * POST call to Cloudant database API
+    * @param jsonString
+    * @return
+    */
+  def post(jsonString: String): Future[String] = {
```

Comment:
That the cloudant client does a post/get is internal details which the calling code does not have anything to do with, I think you should raise the abstraction level one step and call these `save` and `load`. (Imagine the corresponding if the JDBC API would say `sendBytesOverNetworkWithTcp` rather than `executeQuery`)

I still think that accepting strings and producing strings is the wrong abstraction, but one step at a time I guess. 

If it ends up with wanting to avoid parsing the JSON (which I think would be the right thing) for performance reasons I would still introduce a new class denoting that it is not any string but raw json, in that case it may make sense to _never_ even parse it into a string but keep it as bytes. Something like `case class RawJson(rawData: ByteString)`.